src/test/java/uk/gov/hmcts/reform/finrem/notifications/service/EmailServiceTest.java

```diff
@@ -280,4 +282,31 @@ public void shouldBuildTemplateVarsForConsented() {
         assertNull(returnedTemplateVars.get("courtName"));
         assertNull(returnedTemplateVars.get("courtEmail"));
```

Comment:
Be good to add to existing tests to assert that general body data isn't being set whatsoever:
assertNull(returnedTemplateVars.get("generalEmailBody"));

Reply:
added to the consented and contested tests